src/content/docs/paper/dev/api/particles.mdx

```diff
@@ -0,0 +1,319 @@
+(of the preset 24) will be chosen for each spawned particle.
+- If set to `0`, the `offsetX` argument will be used as an addition to a **HSB (hue, saturation, brightness)** color format's
+hue value, which has a starting point at a green color.
+
+Example:
+```java
+someWorld.spawnParticle(Particle.NOTE, someLocation, 0, 0.4f, 0, 0);
+```
+
+:::tip
+To achieve the Vanilla note particle colors you must set the offsetX to a fraction of 24.
+
+Example:
+```java
+someWorld.spawnParticle(Particle.NOTE, someLocation, 0, 2.0f/24.0f, 0, 0);
+```
+:::
+
+### Tinted leaves
+The `TINTED_LEAVES` particle is really simple to spawn, as it only requires a [`Color`](jd:paper:org.bukkit.Color) object
+to be passed as the `data` argument.
+
+Example of spawning 10 orange leaves in a 4x4 area:
+```java
+someWorld.spawnParticle(Particle.TINTED_LEAVES, someLocation, 10, 2, 0, 2, Color.fromRGB(191, 87, 0));
+```
+
+## Converging particles
+As the name implies, this type of particle converges to a single point (location), which in this case is the supplied location.
+
+The particles have two different behaviors based on the `count` argument:
+- If set to a positive number, offset arguments are used normally,
+- If set to `0`, the offset arguments will be used for a relative spawn location from the supplied location/coordinates.
+The particle will then travel from this relative location to the supplied location in a curve.
+
+An example where an enchantment particle will spawn at `someLocation.clone().add(2,0,2)` and travel to `someLocation`:
+```java
+someWorld.spawnParticle(Particle.ENCHANT, someLocation, 0, 2, 0, 2);
+```
+
+### List of converging particles
+<details>
+<summary>Show list</summary>
+- ENCHANT,
+- NAUTILUS,
+- PORTAL.
+</details>
+
+## Material particles
+### BlockData
+To spawn particles that require `BlockData`, simply put `BlockData` as it's `data` argument.
+
+Example:
+```java
+someWorld.spawnParticle(Particle.BLOCK_CRUMBLE, someLocation, 4, BlockType.GLOWSTONE.createBlockData());
+```
+
+:::note
+This guide uses [`BlockType.createBlockData()`](jd:paper:org.bukkit.block.BlockType#createBlockData()). While using
+[`Material.createBlockData()`](jd:paper:org.bukkit.Material#createBlockData()) or
+[`Bukkit.createBlockData(Material)`](jd:paper:org.bukkit.Bukkit#createBlockData(org.bukkit.Material)) yields the same result,
+they are considered **legacy**.
+:::
+
+:::tip
+The `BLOCK` particle is a [directional particle](#directional-particles).
+
+In this case **velocity matters** a lot. A higher velocity will ensure the same general direction of the particle, while a lower one will
+result in a more random direction.
+
+To achieve this, the effective velocity vector's length should be high (around 10 is fine).
+:::
+
+### ItemStack
+To spawn particles that require an `ItemStack`, simply put an `ItemStack` as it's `data` argument.
+
+Example:
+```java
+someWorld.spawnParticle(Particle.ITEM, someLocation, 4, ItemType.DIAMOND_PICKAXE.createItemStack());
```

Comment:
Don't use createItemStack, the typed interface might be gone at some point once ItemMeta is replaced and that might encourage them to use that (ItemStack#of for now).